docs/components/best-practices/modeling/choosing-the-resource-binding-type.md

```diff
@@ -21,13 +21,52 @@ For example, for a call activity this would be the version of the called process
-| `deployment` | <p>Resolves to the specific version of the target resource that was **deployed together** with the currently running version of the process in the **same deployment**.</p><p><ul><li><p>This option ensures predictable behavior by tying the two versions together, and allows you to deploy future versions of the target resource without disrupting ongoing process instances.</p></li><li><p>This option is ideal for self-contained projects without external or shared dependencies.</p></li><li><p>To use the `deployment` binding option, create and deploy a [process application in Web Modeler](/docs/components/modeler/web-modeler/process-applications.md#deploy-and-run-a-process-application), or deploy multiple resources together via the [Zeebe API](/docs/apis-tools/zeebe-api/gateway-service.md#deployresource-rpc).</p></li></ul></p> |
+<table>
+  <thead>
+    <tr>
+      <th align="left">Type</th>
+      <th align="left">Description</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td><code>latest</code></td>
+      <td>
+        <p>Resolves to the <strong>latest deployed version</strong> of the target resource at the moment the process element is activated.</p>
+        <ul>
+          <li>
+            <p>
+              This option makes it easy to change the target resource at runtime by deploying a new version of it,
+              which allows for fast iterations during development and easy hot fixes in production:
+            </p>
+            <ul>
+              <li><p>Process instances that have not yet reached the process element will directly use the newly deployed version of the target resource once the element gets activated.</p></li>
+              <li><p>If the process element has already been reached, <a href="../../../concepts/process-instance-modification">process instance modification</a> can be used to re-activate it and thereby re-link it to the newly deployed version of the target resource.</p></li>
+            </ul>
+          </li>
+          <li><p>Be aware that using <code>latest</code> can lead to unexpected behavior if you deploy a new version of the target resource without ensuring backwards compatibility with every deployed process that depends on it.</p></li>
+          <li><p>Therefore, it might not be suited for production environments that require stability and a predictable behavior.</p></li>
```

Comment:
```suggestion
          <li><p>Be aware that using <code>latest</code> can lead to unexpected behavior if you deploy a new version of the target resource without ensuring backwards compatibility with every deployed process that depends on it.</p><p>This might not be suited for production environments that require stability and a predictable behavior.</p></li>
```